dmutils/apiclient/data.py

```diff
@@ -173,7 +173,12 @@ def update_user_password(self, user_id, new_password):
                         user_id, e.status_code)
             return False
 
-    def update_user(self, user_id, locked=None, active=None):
+    def update_user(self,
+                    user_id,
+                    locked=None,
+                    active=None,
+                    role=None,
+                    supplier_id=None):
```

Comment:
personal preference,often we go over pep8 line length, so we get some on one line and some on the next. I think thats espcially hard to read. Always wrote them like this in JVM languages when the param list got more than a couple.

I find them easier to read, and do it when I'm meddling in the code.....